packages/rx_bloc_cli/README.md

```diff
@@ -0,0 +1,14 @@
+# rx_bloc_cli
+
+Rx Bloc CLI
+
+## Getting Started
+
+This project is a starting point for a Dart
```

Comment:
@DDavidPrime please describe step-by-step instructions about the process of development

Reply:
As discussed all details should be described in the dedicated MD file (ref: [CONTRIBUTING.md](https://github.com/dart-lang/pub-dev/blob/master/CONTRIBUTING.md)) with a reference to the main README.MD